### PR TITLE
Fixed a few small bugs in the English translation

### PR DESCRIPTION
**Pull request by ellenkoenig**
👋  @davhin , @balaklava & @kirel :)

### CODE REVIEW

File: content/apply/index.en.md [hunk 1]
```diff
@@ -17,41 +17,41 @@ We help NGOs to make use of their data by connecting them with volunteer data sc
 <div id="apply--what">
     <ul>
         <li>
-            <a href="{{< ref "datenanalyse-fuer-den-guten-zweck#zielgruppen-verstehen" >}}">
+            <a href="{{< ref "datenanalyse-fuer-den-guten-zweck#zielgruppen-verstehen" lang="en">}}">
```
Comment (kirel) [inline content/apply/index.en.md]: Why is this needed?

Comment (ellenkoenig) [reply content/apply/index.en.md]: I assumed that we eventually want to translate the use cases. Otherwise, what is the point to link them from here?

Comment (kirel) [reply content/apply/index.en.md]: Yep, but there are already files for both translations. It's just german in both versions.

Comment (kirel) [reply content/apply/index.en.md]: See https://github.com/dssg-berlin/dssg-berlin.org/tree/master/content/blog

Comment (ellenkoenig) [reply content/apply/index.en.md]: Yes. Please see my comment in Slack. Maybe I misunderstood how the translations work

**Comment by kirel** [comment]
Ok, hugos i18n is a bit stupid. The best approach (it seems to me) is not having any language pointers but faking always all content available in all languages (.en.md files) - just sometimes it's not yet translated german content.

File: content/apply/index.en.md (hunk 1 above)
Comment (kirel) [reply content/apply/index.en.md]: Yes, not your fault. It's confusing.

**Comment by kirel** [comment]
I'll merge and remove the language pointers. Thank you!

**Comment by ellenkoenig** [comment]
Ah, thanks. I was just about to remove them myself.
